IPython/core/page.py

```diff
@@ -308,8 +308,8 @@ def get_pager_cmd(pager_cmd=None):
         except:
             pager_cmd = default_pager_cmd
     
-    if pager_cmd == 'less' and '-r' not in os.environ.get('LESS', ''):
-        pager_cmd += ' -r'
+    if pager_cmd == 'less' and not os.environ.get('LESS', ''):
```

Comment:
Should this be `'-R' not in ...` - we don't want to go back to not setting it if someone has some entirely unrelated flags in `$LESS`.

Other than that, this looks good, thanks.